mne/epochs.py

```diff
@@ -721,7 +721,7 @@ def ch_names(self):
 
     def plot(self, epoch_idx=None, picks=None, scalings=None,
              title_str='#%003i', show=True, block=False, n_epochs=20,
-             n_channels=20, title=None, trellis=True):
+             n_channels=20, title=None):
```

Comment:
you need to remove the argument `epoch_idx` too, no? Not sure if any of the other arguments need to be removed as well but I think `title_str` might be one of them